### DISCUSSION TITLE
Menu with categories and item descriptions

### DISCUSSION THREAD

**Discussion by pamelahu** [twilio-labs/paste#2423] [Q&A] [answered]
I'm creating a dropdown menu with 7 options that all require descriptions. To make this easier to read, I've added categories (I adjusted the padding to make the groupings more clear but I got feedback that it was still difficult to see the groupings) and then I added a separator.

1. Do these variations still adhere to Paste guidelines?
2. Do y'all have suggestions for ways to display this information?

Dropdown menu with 7 options:
![Screen Shot 2022-05-20 at 11 44 58 AM](https://user-images.githubusercontent.com/18605252/169564400-a689ad5b-bc95-4b73-84ed-0da35d2f9f1e.png)

Dropdown menu with grouping:
![Screen Shot 2022-05-20 at 11 42 21 AM](https://user-images.githubusercontent.com/18605252/169563933-464628f4-c466-4e74-a1dc-1cf61e98acfb.png)

Dropdown menu with grouping and separator:
![Screen Shot 2022-05-20 at 11 42 30 AM](https://user-images.githubusercontent.com/18605252/169563963-307d0463-4913-4fea-9b30-954edc5d0899.png)

**Comment by nkrantz** [comment]
Hey @pamelahu - could you share a Figma link or screenshot of the entire layout so we have a little more context to advise here? Also wondering whether you've gotten any feedback on the third screenshot you shared (since you added the separators). Thanks!

**Comment by pamelahu** [comment]
@nkrantz [Figma link](https://www.figma.com/file/MSxq20XYGrU2Yr1j2YwUJo/%F0%9F%9F%A2-2022-Q2-Expand-Eligibility?node-id=30%3A31946)

The only feedback I've gotten on the third screen is from my PM saying it was easier for him to read!

**Comment by nkrantz** [comment, marked as answer]
@pamelahu thank you for the context and the Figma link! Since the component is in its own Disclosure, my recommendation here would be to try out a Radio Group instead of a Combobox. That way, you won't be constrained at all by the max height of the listbox, and you'll give the user space to read all of the options on the page without having to scroll within the listbox. We'd suggest waiting on the 3rd option (Combobox with Separators) because the DSYS team is currently doing some design exploration on best practices there. 

Let us know if the Radio Group fits your needs once you've tried it out in your layout!
